pybleau/app/plotting/tests/test_dataframe_plot_factories.py

```diff
@@ -380,7 +380,8 @@ def test_multi_line_1_curve_plot(self):
         self.assert_renderers_have_distinct_colors(plot)
 
     def test_multi_line_2_curve_plot(self):
-        renderer_styles = [LineRendererStyle(), LineRendererStyle()]
+        renderer_styles = [LineRendererStyle(color="blue"),
+                           LineRendererStyle(color="red")]
```

Comment:
The point of this test was that you didn't have to specify different colors, they would be generated by the code. Did we lose that with this version?

Reply:
Yes, we did lose this when there is more than one renderer. The renderers should be created with a color (or have it set afterwards) if they are to be different. It is the only way we can control what the colors are. I am fine with removing the last assert (`assert_renderers_have_distinct_colors`), now, and just having the `assert_valid_plot` test.

> The point of this test was that you didn't have to specify different colors, they would be generated by the code. Did we lose that with this version?

Reply:
Thanks for clarifying. I still agree it's the right thing to do. I agree that then, we should then remove the call to `assert_renderers_have_distinct_colors`.